libs/logic-apps-shared/src/parsers/lib/common/__test__/schemaprocessor.spec.ts

```diff
@@ -1110,4 +1110,105 @@ describe('SchemaProcessor Tests', () => {
       })
     );
   });
+
+  it('should be able to process multiple schema types', () => {
+    const schema = {
+      properties: {
+        description: {
+          type: ['string', 'null'],
+        },
+        execarray: {
+          items: {
+            properties: {
+              name: {
+                type: ['array', 'string'],
+              },
+              param1: {
+                type: 'string',
+              },
+            },
+            type: 'object',
+          },
+          type: 'array',
+        },
+        mode: {
+          type: 'string',
+        },
+      },
+      required: ['description', 'mode', 'execarray'],
+      type: 'object',
+    };
+
+    const options = {
+      prefix: '',
+      keyPrefix: 'body.$',
+      isInputSchema: true,
+      expandArrayOutputs: true,
+      required: true,
+    };
+    const parameters = new SchemaProcessor(options).getSchemaProperties(schema);
+    expect(parameters.length).toBe(8);
+    expect(parameters[0]).toEqual(
+      expect.objectContaining({
+        key: 'body.$.description',
+        name: 'description',
+        type: 'string',
+      })
+    );
+
+    expect(parameters[1]).toEqual(
+      expect.objectContaining({
+        key: 'body.$.description',
+        name: 'description',
+        type: 'null',
+      })
+    );
```

Comment:
are you able to see these two inputs in teh UI, is the parameterId distinguishing it or how is this rendered"

Reply:
You don't see 2 inputs in the UI, but it uses both the validation